XML/Other/ReplaceableTestParameters.xml

```diff
@@ -398,11 +398,11 @@ Scenario 2 : You need to run all the performance tests quickly.
 	</Parameter>
 	<Parameter>
 		<ReplaceThis>LIS_CURRENT_URL</ReplaceThis>
-		<ReplaceWith>http://download.microsoft.com/download/6/8/F/68FE11B8-FAA4-4F8D-8C7D-74DA7F2CFC8C/lis-rpms-4.2.8.tar.gz</ReplaceWith>
+		<ReplaceWith>http://download.microsoft.com/download/6/8/F/68FE11B8-FAA4-4F8D-8C7D-74DA7F2CFC8C/lis-rpms-4.3.3.tar.gz</ReplaceWith>
```

Comment:
please test this with https://aka.ms/lis

Reply:
And this can be CustomParameter in run-time

Reply:
@chvalean  CustomLIS parameter accepts .tar.gz links at the moment. 
aka.ms is not supported but will add this in next iteration. @sireeshapotnuru will work on this.
@juhlee-microsoft It is already part of CustomParameter and we can pass it runtime.